db/LousListScript.md

```diff
@@ -0,0 +1,18 @@
+# Lou's List Script
+[UVA Lou's List](louslist.org) links individual courses to tCF course pages, and it requires a table mapping course IDs to the IDs in our URLs (mnemonic, number, database id). They are then fetched at https://thecourseforum.com/course/[course_id].
+
+Run the following commands to generate a table for Lou's List using our database:
+
+0. Make sure you have your database set up correctly. It is likely that you ran `docker-compose down` to start with a clean slate.
+
+1. Open the psql terminal in the tCF database Docker container with this command*:
```

Comment:
Technically after making it a oneliner we're not "opening the psql terminal" anymore, as far as I know

Reply:
Good catch! Fixed in https://github.com/thecourseforum/theCourseForum2/pull/237/commits/6d71c9265e5ae72ff4aebc7cbad263c729e2b81d